pkg/syncer/azure.go

```diff
@@ -191,6 +191,10 @@ func (a *AzureSyncer) Sync() ([]userv1.Group, error) {
 
 		groupName := group.DisplayName
 
+		if groupName == nil {
```

Comment:
Can you add a warning message similar to https://github.com/redhat-cop/group-sync-operator/blob/86d10d185a2f349fc5b2fb6313f513b35c7a68bc/pkg/syncer/azure.go#L259

Reply:
Added the warning - please review.